src/test/scala/edu/ie3/simona/model/participant/PvModelSpec.scala

```diff
@@ -0,0 +1,802 @@
+      }
+    }
+
+    "calculate the angle of incidence thetaG correctly" in {
+      val testCases = Table(
+        (
+          "latitudeDeg",
+          "deltaDeg",
+          "omegaDeg",
+          "gammaEDeg",
+          "alphaEDeg",
+          "thetaGOut",
+        ),
+        (43d, -14d, -22.5d, 45d, 15d,
+          35.176193345578606393727080835951995075234213360724d), // Duffie
+        (
+          51.516667d,
+          +18.4557514d,
+          -15.00225713d,
+          30d,
+          +0d,
+          14.420271449960715d,
+        ), // Iqbal
+        (
+          51.516667d,
+          +18.4557514d,
+          -15.00225713d,
+          90d,
+          +0d,
+          58.65287310017624d,
+        ), // Iqbal
+        (
+          35.0d,
+          +23.2320597d,
+          +30.00053311d,
+          45d,
+          10d,
+          39.62841449023577d,
+        ), // Kalogirou - Solar Energy Engineering Example 2.7  ISBN 978-0-12-374501-9; DOI https://doi.org/10.1016/B978-0-12-374501-9.X0001-5
+        (
+          35.0d,
+          +23.2320597d,
+          +30.00053311d,
+          45d,
+          90d,
+          18.946300807438607d,
+        ), // Kalogirou - Solar Energy Engineering Example 2.7 changed to 90° panel azimuth to WEST
+        (
+          35.0d,
+          +23.2320597d,
+          +74.648850625d,
+          45d,
+          90d,
+          21.95480347380729d,
+        ), // Kalogirou - Solar Energy Engineering Example 2.7  90° panel azimuth to WEST at 17:00
+        (
+          35.0d,
+          +23.2320597d,
+          +74.648850625d,
+          45d,
+          -90d,
+          109.00780288303966d,
+        ), // Kalogirou - Solar Energy Engineering Example 2.7  90° panel azimuth to EAST at 17:00
+        (
+          27.96d,
+          -17.51d,
+          -11.1d,
+          30d,
+          +10d,
+          22.384603601536398d,
+        ), // Goswami Priciples of Solar Engineering Example 2.7a
+        (
+          -35.3d,
+          -17.51d,
+          -4.2d,
+          30d,
+          +170d,
+          14.882390116876563d,
+        ), // Goswami Priciples of Solar Engineering Example 2.7b
+      )
+
+      // Calculate the angle of incidence of beam radiation on a surface located at a Latitude
+      // at a certain hour angle (solar time) on a given declination (date) if the surface
+      // is tilted by a certain slope from the horizontal and pointed to a certain panel azimuth
+      // west of south.
+      forAll(testCases) {
+        (latitudeDeg, deltaDeg, omegaDeg, gammaEDeg, alphaEDeg, thetaGOut) =>
+          // When
+          val thetaG = pvModel.calcAngleOfIncidenceThetaG(
+            Degrees(deltaDeg),
+            Degrees(latitudeDeg),
+            Degrees(gammaEDeg),
+            Degrees(alphaEDeg),
+            Degrees(omegaDeg),
+          )
+
+          // Then
+          thetaG.toDegrees shouldEqual thetaGOut +- 1e-10
+      }
+    }
+
+    "deliver equal angles of incidence for special case" in {
```

Comment:
Can we find a more expressive title here?

Reply:
What do you think of this title?
"deliver equal angles of incidence at different latitudes adjusted by different slope angles"

Reply:
Since the angle of incidence is not always equal over the tests, i would like to avoid the word equal here. But apart from that I like that one.
So maybe `deliver correct angles of incidence at different latitudes and slope angles` ?

Great that this test combines now content which have been in multiple tests before. Maybe we can include some ScalaDoc here giving some context what is covered here. Like the information given in the old test names e.g.: `Testing the equality of zenith angle of a horizontal surface and thetaG of a sloped surface` and maybe the sources as well: e.g. `Iqbal Figure 1.6.2...`

Reply:
I agree with your idea for the title, Daniel

Reply:
What exactly do you mean by "ScalaDoc"? In the function "deliver correct angles of incidence at different latitudes and slope angles", the information from the old test is preserved in a comment. 
Regarding the sources, I copied them from the old test (wherever possible), however, there are some cases with unknown source (for example in the old "calculate Rb..." function)

Reply:
I added ScalaDoc whenever there were multiple lines of comments. I also added one reference to the Duffie example. Please let me know what you think